neo4j/tests/conftest.py

```diff
@@ -9,19 +9,41 @@
 
 @pytest.fixture(scope='session')
 def dd_environment(instance):
+    NEO4J_IMAGE = 'neo4j:4.4.18-enterprise'
     with docker_run(
-        os.path.join(common.HERE, 'docker', 'docker-compose.yaml'),
-        env_vars={'NEO4J_IMAGE': f'neo4j:{common.NEO4J_VERSION}-enterprise'},
+        os.path.join(common.HERE, 'docker', 'docker-compose_v4.yaml'),
+        env_vars={'NEO4J_IMAGE': f'{NEO4J_IMAGE}'},
         log_patterns=['Remote interface available at'],
         endpoints=[common.METRICS_URL],
     ):
         yield instance
 
 
+@pytest.fixture(scope='session')
+def dd_environment_v5(instance_neo4j5):
+    NEO4J_IMAGE = 'neo4j:enterprise'
+    with docker_run(
+        os.path.join(common.HERE, 'docker', 'docker-compose_v5.yaml'),
+        env_vars={'NEO4J_IMAGE': f'{NEO4J_IMAGE}'},
+        log_patterns=['Remote interface available at'],
+        endpoints=[common.METRICS_URL],
+    ):
+        yield instance_neo4j5
+
+
 @pytest.fixture(scope='session')
 def instance():
```

Comment:
Nit: Your fixtures `instance()` and `instance_neo4j5()` are the same.

Reply:
Thanks for noticing that. Fixed that in the latest update.